.github/workflows/notify-triager.yml

```diff
@@ -11,6 +11,23 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4.1.1
 
+      - name: Get commit message
+        id: commit-message
+        run: |
+          commit_message=$(git log --format=%B -n 1 ${{ github.event.pull_request.head.sha }})
+          echo "Commit message: $commit_message" >> $GITHUB_OUTPUT
```

Comment:
The syntax is wrong, as it needs to be in the form `echo "{name}={value}" >> $GITHUB_OUTPUT` where the value can only be accessed by `steps.commit-message.outputs.commit_message` only if `{name}` is `commit_message`

```suggestion
          echo "commit_message=$commit_message" >> $GITHUB_OUTPUT
```